apps/website/src/pages/docs/expandableblock.mdx

```diff
@@ -9,67 +9,83 @@ group: core
+  <AllExamples.ExpandableBlockAccordionExample client:load />
+</LiveExample>
 
-Expandable rows resemble normal table rows that can be expanded to reveal nested data. An expandable row could optionally be selectable. Because expandable rows are part of a table, the rows must sort based on the table column sort applied. This means expandable rows can not re-arranged, the user must interact with the table header to re-arrange.
+Expandable blocks can also stack on top of each other in an accordion to make large sets of information easier to read
 
-Expandable rows should not be confused with a tree hierarchy. Unlike trees, they follow a single branch pattern. If you have trouble telling the two apart, the Expandable block VS Hierarchy tree section explains what makes them different.
+### With status
 
-There can be several child elements nested in an expandable row. These indented elements all coexist on the same level without further indentation. If you require more levels, a tree hierarchy is a better pattern for this interaction.
+<LiveExample src='ExpandableBlock.status.tsx'>
+  <AllExamples.ExpandableBlockStatusExample client:load />
+</LiveExample>
 
-The child elements don’t reflect the column titles. They are expected to retain the same properties as the parent row (in the example below, the owner). It’s unnecessary to repeat the properties for the child items because of this, so do not include them in the expanded row area.
+An expandable block could display status, an icon, or other helpful information at a glance right aligned within the block header.
 
-Child items are not interchangeable between rows, e.g. you cannot drag a child item from row A and drag it in row B like you would move files to a folder.
+### Small
 
-### Expandable blocks
+<LiveExample src='ExpandableBlock.small.tsx'>
+  <AllExamples.ExpandableBlockSmallExample client:load />
+</LiveExample>
 
-The main block has a label reflecting its content. On click, it expands to reveal more content.
+An expandable block can be one of two sizes: default and small
 
-An expandable block could display status, a total count, or other helpful information at a glance right aligned within the block header. You may also optionally display descriptor text underneath the title.
+### Borderless
 
-Expandable blocks don’t always need to be sorted alphabetically, unlike an expandable row which would be sorted based on the table column sort. You may optionally allow expandable blocks to be re-ordered by the user.
+<LiveExample src='ExpandableBlock.borderless.tsx'>
+  <AllExamples.ExpandableBlockBorderlessExample client:load />
+</LiveExample>
 
-The visual appearance of an expandable block is different from an expandable row. An expandable block can not be selected / checked like an expandable row can, therefore it is visually styled differently.
+The borderless expandable block is useful in tighter areas where using a default expandable block would visibly cause too many blocks.
```

Comment:
styleType prop would be useful to mention here

Reply:
This sentence has been added:

"The borderless expandable block is achieved by using the `styleType` prop."